### PR TITLE
Catch up commits from day 1-3 assignments.

### PR DESCRIPTION
**Pull request by kriskuchinka**

### CODE REVIEW

File: codingtovictory/day3/takeHomeChallenge/about.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="styles/styles.css">
+</head>
+<body>
+    <nav>
+        <ul><a href="index.html">Home</a></ul>
```
Comment (victorSauceda) [inline codingtovictory/day3/takeHomeChallenge/about.html:12]: You don't need `ul` around each of these. You should have one and each item should be wrapped around `<li>`. Example

```
<ul>
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ul>
```

File: codingtovictory/day3/takeHomeChallenge/contact.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <link rel="stylesheet" href="styles/styles.css">
+</head>
+<body>
+    <nav>
+        <ul><a href="index.html">Home</a></ul>
+        <ul><a href="merchandise.html">Merchandise</a></ul>
```
Comment (victorSauceda) [inline codingtovictory/day3/takeHomeChallenge/contact.html:13]: Same here

File: codingtovictory/day3/takeHomeChallenge/about.html (hunk 1 above)
Comment (kriskuchinka) [reply codingtovictory/day3/takeHomeChallenge/about.html:12]: Thanks for pointing this out. It's funny that it worked the wrong way. I have corrected it on all the pages of this assignment.

File: codingtovictory/day3/takeHomeChallenge/contact.html (hunk 2 above)
Comment (kriskuchinka) [reply codingtovictory/day3/takeHomeChallenge/contact.html:13]: Thanks for pointing this out. It's funny that it worked the wrong way. I have corrected it on all the pages of this assignment.

**Pull request merged**